tomviz/ModuleContour.cxx

```diff
@@ -410,6 +430,8 @@ std::string ModuleContour::getStringForProxy(vtkSMProxy* proxy)
     return "Representation";
   } else if (proxy == this->ResampleFilter.Get()) {
     return "Resample";
+  } else if (proxy == this->PointDataToCellDataFilter.Get()) {
+    return "Resample";
```

Comment:
According to the documentation for `Module::getStringForProxy(vtkSMProxy* proxy)`, the string returned should be unique for the various internal proxies, so this should probably return "PointDataToCellData" to match the logic in `ModuleContour::getProxyForString(const std::string& str)` below.